schemas/contribution.json

```diff
@@ -43,12 +43,15 @@
       "title": "Kind",
       "description": "Type/category of contribution",
       "enum": [
+        "bureaucracy",
```

Comment:
"bureaucracy" sounds so negative. How about just "administration" instead?

Reply:
It was intended to sound negative. :)

As a decentralized co-operative, there shouldn't actually be any administration to be done in the end IMO. So anything that has to be done is basically to satisfy legacy world requirements the way I see it.

But I'm also fine with changing it. As this PR has been merged, you will just have to open a new one if you care about it.